index.html

```diff
@@ -1,12 +1,374 @@
 <!DOCTYPE html>
 <html lang="ru">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="/css/style.css" />
     <title>HTML Academy: Девайс</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
+    <div class="wrapper-page">
+      <h1 class="visualy-hidden">Девайс</h1>
+      <header class="header-page">
+        <div class="user-nav">
```

Comment:
Если делать совсем корректно, то блок так и нужно называть - блок, контейнер, например 
```suggestion
        <div class="user-nav-container">
```